packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts

```diff
@@ -68,6 +68,10 @@ export class DocsIndexer extends Entity {
 
   setupListener() {
     this.workspaceEngine.doc.storage.eventBus.on(event => {
+      if (event.docId.startsWith('db$')) {
```

Comment:
May be better to maintain `dbDoc.id` validation together with initialization?
```ts
import { isDBDocId } from "xxx"; // where db doc id is initialized

if (isDBDocId(event.docId) {}
```

Reply:
and could we make this condition a filter or in/ex-clude option?